app/controllers/verify/finance_controller.rb

```diff
@@ -49,7 +49,7 @@ def step_params
     end
 
     def confirm_step_needed
-      redirect_to verify_phone_path if idv_session.financials_confirmation == true
+      redirect_to verify_address_path if idv_session.financials_confirmation == true
```

Comment:
so should this be `_url` too??

so excited to be the world's 🥇  `_url` advocate